### PR TITLE
Fix encoding error in API payload.

### PR DESCRIPTION
**Pull request by rheinardkorf**

### CODE REVIEW

File: src/XWP/Bundle/AuditServerBundle/Extensions/CodeIdentityManager.php [hunk 1]
```diff
@@ -49,8 +49,8 @@ public function getWordPressCodeInfo($auditsFilesDirectory)
         foreach ($plugin_theme_header as $key => $value) {
             if ($value) {
                 $codeInfo['details'][] = array(
-                    'key' => $key,
-                    'value' => $value,
+                    'key' => mb_convert_encoding($key, "UTF-8"),
```
Comment (rheinardkorf) [inline src/XWP/Bundle/AuditServerBundle/Extensions/CodeIdentityManager.php]: @valendesigns this was the source of the error

Comment (valendesigns) [reply src/XWP/Bundle/AuditServerBundle/Extensions/CodeIdentityManager.php]: What key would be causing that in the plugin header

File: src/XWP/Bundle/AuditServerBundle/Extensions/Audits/PhpCsAuditManager.php [hunk 2]
```diff
@@ -136,7 +136,8 @@ public function runAudit(
         $fullReportFilename = $auditsFilesChecksum.'-phpcs-'.$this->auditStandardKey.'-full.'.$options['report'];
         $fullReportPath = $auditsReportsDirectory . '/' . $fullReportFilename;
 
-        $command = "phpcs $stringOptions --report-{$options['report']}=$fullReportPath $auditsFilesDirectory -q";
+        $command = "phpcs $stringOptions --encoding=utf-8 " .
```
Comment (valendesigns) [inline src/XWP/Bundle/AuditServerBundle/Extensions/Audits/PhpCsAuditManager.php]: This should be coming from the config options in the SS and API, I'll create PR's to update them and then we'll remove it from here.

File: src/XWP/Bundle/AuditServerBundle/Extensions/CodeIdentityManager.php (hunk 1 above)
Comment (rheinardkorf) [reply src/XWP/Bundle/AuditServerBundle/Extensions/CodeIdentityManager.php]: just bad encoding... most likely description

**Comment by rheinardkorf** [comment]
changes here looks good

**Pull request merged**